doc/api/util.md

```diff
@@ -666,7 +666,7 @@ Deprecated predecessor of `console.log`.
 
 Deprecated predecessor of `console.log`.
 
-### util._extend(obj)
+### util._extend(targetObj, sourceObj)
```

Comment:
Should be origin and add to keep consistent with source, see https://github.com/nodejs/node/blob/master/lib/util.js#L976